backend/capellacollab/projects/toolmodels/modelsources/git/exceptions.py

```diff
@@ -1,174 +1,99 @@
 # SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
 # SPDX-License-Identifier: Apache-2.0
 
-import dataclasses
+from fastapi import status
 
-import fastapi
-from fastapi import exception_handlers, status
+from capellacollab.core import exceptions as core_exceptions
 
 
-class AccessDeniedError(Exception):
+class AccessDeniedError(core_exceptions.BaseError):
```

Comment:
```suggestion
class AccessDeniedError(core_exceptions.BaseError, metaclass=abc.ABCMeta):
```

The `AccessDeniedError` and `RepositoryNotFoundError` classes are abstract classes.